src/inject/dynamic-theme/inline-style.ts

```diff
@@ -85,6 +85,8 @@ const overrides: Overrides = {
 };
 
 const overridesList = Object.values(overrides);
+const customPropsList = {};
```

Comment:
Maybe it should has some other name? E.g. it looks like a map of Dark Reader's custom properties and real CSS properties.